src/truncate/truncate.scss

```diff
@@ -3,18 +3,22 @@
 // css-truncate will shorten text with an ellipsis.
 
 .css-truncate {
-  // Truncate double target
-  //
-  // css-truncate will shorten text with an ellipsis. The maximum width
-  // of the truncated text can be changed by overriding the max-width
-  // of the .css-truncate-target
+
```

Comment:
The diff is a bit messy with these "double selectors". :weary: Without it's:

```scss
// Both share these properties
.css-truncate-overflow,
.css-truncate-target {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}

// .css-truncate-target has these additional properties
.css-truncate-target {
  display: inline-block;
  max-width: 125px;
  vertical-align: top;
}
```

Reply:
It's also tempting to rename it to something like:

```scss
// Default
.Truncate {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}

// Modifier for inline elements
.Truncate--inline {
  display: inline-block;
  max-width: 125px;
  vertical-align: top;
}
```

Would require some bigger refactor on dotcom.